public/docs/ts/latest/tutorial/toh-pt2.jade

```diff
@@ -105,206 +110,214 @@ code-example(language="bash").
-    This UI pattern is widely known as "master-detail".
-    In our case, the master is the heroes list and the detail is the selected hero.
+    ## Selecting a hero
+    The app now displays a list of heroes as well as a single hero in the details view.
+    The list and the details view are not connected in any way.
+    When the user selects a hero from the list, we want the selected hero to appear in the details view.
+    This UI pattern is known as "master-detail."
+    In this case, the master is the heroes list and the detail is the selected hero.
 
-    Let’s connect the master to the detail through a `selectedHero` component property bound to a click event.
+    Let's connect the master to the detail through a `selectedHero` component property, which is bound to a click event.
 
-    ### Click event
-    We modify the `<li>` by inserting an Angular event binding to its click event.
+    ### Add a click event
+    Modify the `<li>` by inserting an Angular event binding to its click event.
 
   +makeExample('toh-2/ts-snippets/app.component.snippets.pt2.ts', 'selectedHero-click', 'app.component.ts (template excerpt)')
 
   :marked
-    Focus on the event binding
+    Focus on the event binding:
   code-example(format="." language="bash").
     (click)="onSelect(hero)"
   :marked
-    The parenthesis identify the `<li>` element’s  `click` event as the target.
+    The parentheses identify the `<li>` element's  `click` event as the target.
     The expression to the right of the equal sign calls the  `AppComponent` method, `onSelect()`,
     passing the template input variable `hero` as an argument.
-    That’s the same `hero` variable we defined previously in the `ngFor`.
+    That's the same `hero` variable you defined previously in the `ngFor` directive.
   .l-sub-section
     :marked
-      Learn more about Event Binding in the 
+      Learn more about event binding in the 
       [User Input](../guide/user-input.html) and
-      [Templating Syntax](../guide/template-syntax.html#event-binding) chapters.
+      [Templating Syntax](../guide/template-syntax.html#event-binding) pages.
   :marked
     ### Add the click handler
-    Our event binding refers to an `onSelect` method that doesn’t exist yet.
-    We’ll add that method to our component now.
-
-    What should that method do? It should set the component’s selected hero to the hero that the user clicked.
-
-    Our component doesn’t have a “selected hero” yet either. We’ll start there.
+    <!-- CF: (Sorry for the change in comment style; I can't seem to add a // comment
+    anywhere in this section without commenting out the rest of the section and the next section.)
+    This section title is a verb, making it sound like a task, but it's really an introduction;
+    the tasks it discusses aren't performed until the next section. Would it make sense to merge
+    the two sections and maybe change the title to "Add a click handle to expose the selected hero"? -->
```

Comment:
Yes, I think that makes sense. This section really is just a very short intro that could be reduced (because of the conversational style) and then put in the next section.